packages/transforms/snapshot/package.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "@graphql-mesh/snapshot",
+    "version": "0.0.16",
+    "main": "dist/index.js",
+    "license": "MIT",
+    "scripts": {
+      "test": "jest --config=../../../jest.config.js",
+      "build": "tsc"
+    },
+    "peerDependencies": {
+      "graphql": "*"
+    },
+    "devDependencies": {
+      "@types/fs-extra": "8.1.0",
+      "@types/jest": "25.1.4",
+      "@types/md5": "2.1.33",
+      "@types/node": "13.9.4",
+      "graphql": "14.6.0",
+      "jest": "25.1.0",
+      "ts-jest": "25.2.1",
+      "ts-node-dev": "1.0.0-pre.44",
+      "typescript": "3.8.3"
+    },
+    "dependencies": {
+      "@graphql-mesh/types": "0.0.16",
+      "@graphql-toolkit/common": "0.9.9",
+      "fs-extra": "9.0.0",
+      "md5": "2.2.1",
```

Comment:
maybe use `object-hash`? it's simpler and works for everything ;) also in browser